server/service/frontend.go

```diff
@@ -68,6 +70,71 @@ func ServeFrontend(urlPrefix string, sandbox bool, logger log.Logger) http.Handl
 	})
 }
 
+func ServeEndUserEnroll(urlPrefix string, logger log.Logger) http.Handler {
+	herr := func(w http.ResponseWriter, err string) {
+		logger.Log("err", err)
+		http.Error(w, err, http.StatusInternalServerError)
+	}
+
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		writeBrowserSecurityHeaders(w)
+
+		fs := newBinaryFileSystem("/frontend")
```

Comment:
Won't this directory already exist for the name frontend bundle?

Reply:
@gillespi314 could you expand on what do you mean by that? the function creates a new instance of `assetfs.AssetFS` which is just reading assets from the build.

Reply:
Thanks for the clarification. I hadn't looked into what the `newBinaryFileSystem` was doing under the hood. The comment was just meant as a double check in case the constructor was recreating the directory or other potentially destructive activity. Sounds like we're all good here.

Reply:
Thank you! marking as solved.